edx_exams/apps/core/api.py

```diff
@@ -160,6 +170,24 @@ def _calculate_allowed_mins(exam):
 
     return allowed_time_limit_mins
 
+def _emit_TODO_NAME_event(username, content_id):
+    # .. event_implemented_name: TODO
+    STUDENT_REGISTRATION_COMPLETED.send_event(
+        user=UserData(
+            id=1,
+            is_active=True,
+            pii=UserPersonalData(
+                username=username,
+                email='test',
+                name='test',
+            )
+            # data=InstructorCompletionData(
```

Comment:
This was just me commenting what our event data will likely look like. I know that is still in flux and will change, though.